samples/Calculator/package.json

```diff
@@ -12,8 +12,8 @@
   },
```

Comment:
@chiaramooney updating the samples will be a little bit more involved than changing just versions. I can paste over some instructions, but would you mind us doing that as a separate change?

Reply:
Yea definitely. Just did the version change because at first the Calc CI was failing during the `yarn install` but looks like it's break is more involved. Best we fix that in a separate PR.